### PR TITLE
Add cypress for component and e2e testing

### PR DESCRIPTION
**Pull request by 4leite**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,23 +12,95 @@ concurrency:
...
       - name: Package
         run: npm run package
+      - name: Store build
+        uses: actions/upload-artifact@v3
+        with:
+          name: storybook-static
+          path: apps/storybook/storybook-static
+  test:
+    name: Test
+    runs-on: ubuntu-20.04
```
Comment (andrewwh) [inline .github/workflows/ci.yml:41]: What is this fixed to 20.04 and the build is latest?

File: .gitignore [hunk 2]
```diff
@@ -10,4 +10,6 @@ node_modules
 .env
 .next
 dist
-*.tsbuildinfo
\ No newline at end of file
+*.tsbuildinfo
+cypress/screenshots*
+cypress/videos*
```
Comment (andrewwh) [inline .gitignore:15]: Do you need this here if it is also in the storybook directory?

**Review by andrewwh — DISMISSED** [review]
Could you also update the developer documentation (DEVELOP.md) or appropriate sub-documents?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (4leite) [reply .github/workflows/ci.yml:41]: This effects the browser version number so it's good to fix it for the test suite.
We could fix the build as well - but generally the build context for github actions is set to latest. I'm happy to change if you want.

**Review by andrewwh — APPROVED** [review]

**Review by andrewwh — APPROVED** [review]

**Pull request merged**